src/Model/Order/Order.php

```diff
@@ -103,12 +152,12 @@ public function toNative(): array
     }
 
     public static function create(
-        OrderId $id,
-        BusinessName $businessName,
-        BusinessAddress $businessAddress,
-        ContactPerson $contactPerson,
-        OrderIsPaid $isPaid,
-        OrderAmount $amount
+        NonNullOrderId $id,
+        NonNullBusinessName $businessName,
+        NonNullBusinessAddress $businessAddress,
+        NonNullContactPerson $contactPerson,
+        NonNullPaymentStatus $paymentStatus,
+        NonNullOrderAmount $amount
```

Comment:
Similar to my previous comment about the paymentStatus argument

Reply:
It has been updated now.